components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -371,6 +386,11 @@ - (void)viewWillLayoutSubviews {
     }
   } else {
     contentViewFrame.size.height = self.contentViewController.preferredContentSize.height;
+    if ([self isAccessibilityMode]) {
+      contentViewFrame.size.height =
+          MAX(contentViewFrame.size.height,
+              self.presentingViewBounds.size.height - self.topHeaderHeight);
```

Comment:
Why is this only necessary in accessibility mode?

Reply:
same explanation as above